components/brave_wallet/browser/brave_wallet_service.cc

```diff
@@ -352,8 +357,8 @@ bool BraveWalletService::AddUserAsset(mojom::BlockchainTokenPtr token,
   if (network_id.empty())
     return false;
 
-  // Verify input token ID for ERC721.
-  if (token->is_erc721) {
+  bool check_token_id = token->is_erc721 || token->is_erc1155;
```

Comment:
Should it be `token->is_erc721 || (token->is_erc1155 && token->is_nft)` ?

Reply:
I don't think so since I believe all ERC1155 tokens will have a token ID